CalculatorEngine.java

```diff
@@ -0,0 +1,78 @@
+import java.util.Scanner;
+
+public class CalculatorEngine {
+    private int firstArabicNumber;
+    private int secondArabicNumber;
+    private Scanner input = new Scanner(System.in);
+    private Calculations calculations = new Calculations();
+    private RomanToArabicNumbersChanger romanToArabicNumbersChanger = new RomanToArabicNumbersChanger();
```

Comment:
creating object should be injected from class above (Get familiar with dependency injection: https://www.youtube.com/watch?v=kNWwEVJLe48)